go.mod

```diff
@@ -4,8 +4,10 @@ go 1.15
 
 require (
 	github.com/go-resty/resty/v2 v2.1.1-0.20191201195748-d7b97669fe48
+	github.com/google/uuid v1.2.0
 	github.com/gruntwork-io/terratest v0.30.15
 	github.com/k8ssandra/cass-operator v1.7.0
+	github.com/k8ssandra/reaper-client-go v0.3.1-0.20210604120135-4249ab61881c
```

Comment:
This is temporary as this PR is currently depending on the reaper client PR:

https://github.com/k8ssandra/reaper-client-go/pull/6